slave.mk

```diff
@@ -168,6 +168,7 @@ endif
 
 ifeq ($(SONIC_INCLUDE_SYSTEM_GNMI),y)
 INCLUDE_SYSTEM_GNMI = y
+INCLUDE_SYSTEM_TELEMETRY = n
```

Comment:
That is a good point.  I suppose someone could run a gnmi server through the docker-sonic-gnmi container for one purpose and also, on a different port, run a second gnmi server through docker-sonic-telemetry.  If you have guarantees that the clients will not step on each other (e.g. run one gnmi server in read-only mode) it could work.  Probably not something most people would be interested in, but why add the restriction to prevent it.

We have not had problems where we inadvertently enabled telemetry when we only wanted gnmi, so this change is probably not required.